lib/options.h

```diff
@@ -33,6 +33,7 @@ static OPTION config_options_default[] = {
 {"serv-type",		OT_STR, ST_UNDEF, NULL},
 {"serv-auth-type",	OT_STR, ST_UNDEF, NULL}, /* alias for serv-type */
 {"namespace",		OT_STR, ST_UNDEF, NULL}, 
+{"ipv6_use_non_temp_addr",	OT_STR, ST_UNDEF, NULL}, 
```

Comment:
maybe we can get rid of the ipv6 prefix, and use hyphens instead of underscores; simply `use-non-temp-addr`. Then we only document that this is an ipv6-only option.